config/config.ts

```diff
@@ -105,11 +105,12 @@ const config: Configuration = {
                             url: 'https://flybywirecdn.com/addons/a32nx/master',
                             alternativeUrls: [
                                 'external/a32nx/master',
-                                // move old experimental users over to dev
-                                'https://cdn.flybywiresim.com/addons/a32nx/cfbw-cap',
-                                'https://cdn.flybywiresim.com/addons/a32nx/cfbw',
                                 // move bunnycdn users to cloudflare
                                 'https://cdn.flybywiresim.com/addons/a32nx/master',
+				// move exp users to dev
+				'https://flybywirecdn.com/addons/a32nx/experimental',
```

Comment:
```suggestion
				'external/a32nx/experimental',
				'https://flybywirecdn.com/addons/a32nx/experimental',
```

Reply:
There is some bad tab/space whitespacing on this line (GitHub renders a different indent).

Reply:
Both addressed 👍